lib/evss/claims_service.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
 require 'evss/base_service'
+require 'sentry/rescue_evss_errors.rb'
```

Comment:
in older code we were still using requires but now we've switched to using autoloading so this isn't needed anymore if your files are named according to autoloading convention